src/dummy.cpp

```diff
@@ -0,0 +1 @@
+// This file is required so that package installation uses a C++ linker.
```

Comment:
Can you add a link to https://cran.r-project.org/doc/manuals/r-release/R-exts.html#External-C_002b_002b-code in a comment?